refinery/config/settings/base.py

```diff
@@ -670,3 +671,9 @@ def get_setting(name, settings=local_settings, default=None):
 USER_FILES_FACETS = get_setting("USER_FILES_FACETS")
 
 MEDIA_BUCKET = ''  # a placeholder for use in context processor
+
+# Allow JSONSchema to find the JSON pointers to our schema files
+JSON_SCHEMA_FILE_RESOLVER = RefResolver(
+    "file://{}/".format(os.path.join(BASE_DIR, "refinery")),
+    None
+)
```

Comment:
An instance of RefResolver should not be a Django setting. Also, it looks like [base_uri](http://python-jsonschema.readthedocs.io/en/latest/references/#jsonschema.RefResolver) doesn't need to be a file path but could be any URI: something like http://refinery-platform.org to make things simpler.
Finally, a better approach might be to add the [$id keyword](http://json-schema.org/latest/json-schema-core.html#id-keyword) to the schema and then use [from_schema class method](http://python-jsonschema.readthedocs.io/en/latest/references/#jsonschema.RefResolver.from_schema) to create a resolver?